docs/reference/api/connectivity/nfc/NFC.md

```diff
@@ -0,0 +1,110 @@
+##   NFC
+
+NFC stands for Near-Field Communication and is a short-range radio technology. You can use it to enable use cases such as contactless payments, access control or device pairing.
+
+![N Mark][n_mark]
+
+You can find more details on NFC about the [NFC Forum's website](https://nfc-forum.org/what-is-nfc/).
+
+### NFC support in Mbed OS
+
+Historically, NFC encompasses multiple protocols. NFC support in Mbed OS is based on the specifications the [NFC Forum](https://nfc-forum.org/our-work/specifications-and-application-documents/specifications/nfc-forum-technical-specifications/) publishes.
+
+NFC offers three modes:
+
+1. NFC card emulation.
+1. NFC reader or writer.
+1. NFC peer to peer.
+
+Mbed OS supports the card emulation mode, either through the use of a controller or NFC EEPROM.
+
+NFC support is available for the NXP PN512 transceiver. In terms of NFC EEPROMs, a driver for the ST M24SR EEPROM series is available - we also provide a [porting guide](https://os.mbed.com/contributing/connectivity/NFCEEPROMDriver.html) to add support for other NFC EEPROMs.
```

Comment:
Let's remove the first part of this, so what we support doesn't quickly become out of date.